docs/build-html.script

```diff
@@ -0,0 +1,52 @@
+/*
+ * build-html.script
+ * this js-file has funky .script extension to avoid being copied to /out/docs/
+ */
+/*jslint node*/
+"use strict";
+let fs;
+let head;
+let marked;
+let path;
+let tail;
+// require modules
+fs = require("fs");
+marked = require("marked");
```

Comment:
wait, `marked` is a dependency where? (I do realize we were already using the binary, just curious).

Reply:
its a devDependency in docs/package.json

Reply:
Ah, my bad. Thanks.